tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -845,18 +849,56 @@ describe('scalar card', () => {
       flush();
     }));
 
-    it('dispatches metricsSlideoutMenuOpened when edit columns button is clicked', fakeAsync(() => {
+    it('dispatches metricsSlideoutMenuOpened with mode SINGLE when there are no fobs when edit columns button is clicked', fakeAsync(() => {
```

Comment:
Good catch. I originally did check the fob state instead of the ngrx boolean. I forgot to finish cleaning up these tests after I made the change to look at the boolean.

Changed.